src/streaming/utils/CapabilitiesFilter.js

```diff
@@ -162,19 +162,97 @@ function CapabilitiesFilter() {
         }
     }
 
-    function _createVideoConfiguration(rep, codec) {
-        const width = rep.width || null;
-        const height = rep.height || null;
-        const framerate = rep.frameRate || null;
-        const bitrate = rep.bandwidth || null;
+    function _convertHDRColorimetryToConfig(representation) {
+        let cfg = {
+            colorGamut: null,
+            transferFunction: null,
+            isSupported: true
+        };
 
-        return {
-            codec,
-            width,
-            height,
-            framerate,
-            bitrate
+        for (const prop of representation.EssentialProperty || []) {
+
+            // note: MCA does not reflect a parameter related to 'urn:mpeg:mpegB:cicp:VideoFullRangeFlag'
+
+            // translate ColourPrimaries signaling into capability queries
+            if (prop.schemeIdUri == Constants.COLOUR_PRIMARIES_SCHEME_ID_URI && ['1', '5', '6', '7'].includes(prop.value.toString())) {
+                cfg.colorGamut = 'srgb';
```

Comment:
Can we move all strings like `srgb`, `p3` etc to `Constants`

Reply:
_Question:_
Should we use `Constants `in unit tests as well? Having strings there would detect accidental typos in `...\src\streaming\constants\Constants.js`

Reply:
Sounds good, let's keep the strings in the unit tests then or we add unit tests to check the Constants values. The second one might be a bit "overkill"